### PR TITLE
Disabled cache busting when url is in the form 'blob:'

### PR DESCRIPTION
**Pull request by esteban-dosztal**
Pull Request for issue https://github.com/google/shaka-player/issues/395.

### CODE REVIEW

**Comment by joeyparrish** [comment]
@esteban-dosztal, the code change looks fine to me.  Can you please add yourself to CONTRIBUTORS and AUTHORS as well?

**Comment by esteban-dosztal** [comment]
Sorry, forgot that step. Thanks!

File: AUTHORS [hunk 1]
```diff
@@ -27,3 +27,4 @@ Sanborn Hilland <sanbornh@rogers.com>
 TalkTalk Plc <*@talktalkplc.com>
 uStudio Inc. <*@ustudio.com>
 Mattias Wadman <mattias.wadman@gmail.com>
+Esteban Dosztal <edosztal@gmail.com>
```
Comment (joeyparrish) [inline AUTHORS]: Please alphabetize.

File: CONTRIBUTORS [hunk 2]
```diff
@@ -42,3 +42,4 @@ Timothy Drews <tdrews@google.com>
 Vasanth Polipelli <vasanthap@google.com>
 Vignesh Venkatasubramanian <vigneshv@google.com>
 Mattias Wadman <mattias.wadman@gmail.com>
+Esteban Dosztal <edosztal@gmail.com>
```
Comment (joeyparrish) [inline CONTRIBUTORS]: Please alphabetize.

**Comment by esteban-dosztal** [comment]
Hi,

Do you have any idea about if ths PR will be merged into the 1.x version?
Thanks!

**Comment by joeyparrish** [comment]
Sorry, github does not notify us when you make revisions to your PR.  You have to say something in the comments for us to know to check it again.  :-)

I'm just back from vacation, hence the additional 7 days of latency.  I'll merge it today.

**Comment by joeyparrish** [comment]
Merged manually with cosmetic changes.

**Pull request closed without merging**